### PR TITLE
MAIN: Update FSL gear licenses and descriptions

### PR DESCRIPTION
**Pull request by naterichman**

### CODE REVIEW

File: bin/update.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import sys
```
Comment (NPann) [inline bin/update.py:1]: nit: add a little description about what that script is for ?

**Review by NPann — COMMENTED** [review]
I think we actually need to modify the on the manifests folder instead no ? (that the one that are use by the CLI when installing new gear or to render the exchange I think).

File: gears/flywheel/bids-freesurfer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+  "description": "BIDS-Apps/Freesurfer (6.0.1-5) This app implements surface reconstruction using Freesurfer. It reconstructs the surface for each subject individually and then creates a study specific template. In case there are multiple sessions the Freesurfer longitudinal pipeline is used (creating subject specific templates) unless instructed to combine data across sessions.  The current Freesurfer version is based on: freesurfer-Linux-centos6_x86_64-stable-pub-v6.0.0.tar.gz.\n \tLICENSING NOTE: FSL software are owned by Oxford University Innovation and\n license is required for any commercial applications. For commercial licence\n please contact fsl@innovation.ox.ac.uk . For academic use, an academic license\n is required which is available by registering on the FSL website. Any use of\n the software requires that the user obtain the appropriate license. See\n https://fsl.fmrib.ox.ac.uk/fsldownloads_registration for more information.\n",
```
Comment (NPann) [inline gears/flywheel/bids-freesurfer.json:4]: Typo: `fsl@innovation.ox.ac.uk .` -> `fsl@innovation.ox.ac.uk.`

What drove the `\n` ? Some seem to be in weird location to me.

**Review by NPann — COMMENTED** [review]
LGTM but that formatting oddity.

File: gears/flywheel/bids-freesurfer.json (hunk 2 above)
Comment (naterichman) [reply gears/flywheel/bids-freesurfer.json]: Updated!

**Review by NPann — APPROVED** [review]

**Pull request merged**